components/Header/Header.tsx

```diff
@@ -25,7 +25,7 @@ const Header = () => {
       <div className="flex items-center gap-4">
         <Tooltip
           id={"new-conversation-tooltip"}
-          message="New Chat"
+          message={"New Chat"}
```

Comment:
Are the `{}` required?
- what value are they adding?
```suggestion
          message="New Chat"
```